src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -27,6 +28,10 @@ class Vehicle(val id: Int, val wannaDriveInHours: MutableList<Int>) {
         if (!wannaDriveInHours.contains(timestep)) {
             delay -= 1
         }
+
+        if (delay <= 0 && wannaDriveInHours.contains(timestep)){
```

Comment:
Wenn `delay` nicht kleiner-gleich null ist, hat das Fahrzeug noch Verspätung aufzuholen. Dann fuhr es nicht ohne Verspätung?

Finde es etwas schlecht durchschaubar wann ein Fahrzeug den Status gefahren bekommen soll. Ich denke es wäre so:
* Ein Vehicle wird immer so fahren, wie es in der Liste `wannaDriveInHours` steht
* Kommt es zu `TrafficJam` fährt das Fahrzeug auch?!, jedoch wird es `delayed` in diesem `timestep`
* Der Status `delayed` zieht sich durch die `timesteps` durch, bis `delay<=0`. Erst wenn die Verspätung aufgeholt ist, fährt das `Vehicle` nicht mehr verspätet. Also kann `driveWithoutDelay` auch vermeintlich staufreie Zeiten nicht beinhalten, wenn das `Vehicle` Zeit aufholen musste.

Wenn man es anders macht würden einzelne `timesteps` sowohl in der Liste `delayed` als auch `droveWithoutDelay` auftauchen (glaube ich).